src/modules/userMethods.ts

```diff
@@ -206,6 +214,7 @@ export const userMethods = {
                 },
                 supervisor: true,
                 achievements: { include: { achievement: true } },
+                settings: true,
```

Comment:
Hide achievements here, after checking setings

Reply:
prisma can do it?

Reply:
I did it inside this method